README.md

```diff
@@ -1 +1,2 @@
-TinyWebDB
+# Makeroid TinyWebDB Server
+The official Makeroid TinyWebDB Server for you to use only for test purposes.
```

Comment:
Technically is for both testing and production purposes, as to set up one is not an easy action and users won't have time to do so

Reply:
So what do you suggest me to do? Change the phrase from:
"The official Makeroid TinyWebDB Server for you to use only for test purposes." 
to
"The official Makeroid TinyWebDB Server for you to use only for test and production purposes."?